content/rest/reference/gitignore.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Gitignore
+intro: "The Gitignore API fetches all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories."
```

Comment:
It would be more accurate to say that .gitignore is used to ignore files. Also, since the repo is linked in the next paragraph with more details, I would recommend omitting the link here.
```suggestion
intro: "The Gitignore API fetches `.gitignore` templates that can be used to ignore files and directories."
```